guides/integrations-api/general-concepts/request-headers.md

```diff
@@ -22,6 +22,15 @@ Shopware only populates a translatable field if there is an explicit translation
 **Example:** You instruct Shopware to fetch the french translation of a product using the `sw-language-id` header, but there's no french translation for the products name. The resulting field `product.name` will be `null`. When you're building applications, always use `product.translated.[value]`to access translated values, to make sure you will always get a valid translation or fallback value.
 :::
 
+## sw-version-id
+
+Shopware 6 allows developers to create multiple versions of an entity. This has been used, for example, for orders. This allows relations, like documents, to pin a specific state of the entity it relates to. However, the API initially only delivers the data of the most recent record. To tell the API that a specific version should be returned, the header `sw-version-id` must be sent along with the request.
+
+```bash
+POST /api/search/order
+--header 'sw-version-id: 0fa91ce3e96a4bc2be4bd9ce752c3425'
+```
+
```

Comment:
You might want to update your fork since the file is not up 2 date (to avoid a possible conflict)

![image](https://github.com/shopware/docs/assets/8600299/33d11cbd-f23f-472f-bba2-98034797daeb)

Reply:
I've pulled the main branch before making any edits. What exactly is conflicting here?
But just to be sure, I've rebased the commit on top of the latest main branch.
Or should I base it on another branch?

Reply:
![image](https://github.com/shopware/docs/assets/8600299/d550e63c-0bfe-4c85-8c61-f68580bd488a)
Was missing in my github view (there was no expand option at the bottom, maybe a hickup or I missed it) :+1: